pydantic_redis/abstract.py

```diff
@@ -21,6 +21,9 @@ class Config:
         arbitrary_types_allowed = True
         orm_mode = True
 
+def pydantic_serializer(data):
+    if isinstance(data, _AbstractModel):
+        return data.json()
```

Comment:
Let's explicitly raise a TypeError here in case the `data` argument is not `_AbstractModel`
Otherwise it will fail silently (and put a 'null') in redis. 

To show this, try changing line 26 to `return None` (which it does for anything other than _AbstractModel instances), run your sample app and check redis.

Reply:
```suggestion
        return data.json()
    raise TypeError(f"Type {type(data)} cannot be serialized")
```